src/index.js

```diff
@@ -11,7 +12,7 @@ library.add(fab, faCheckSquare, faCoffee, faRocket, faPiggyBank)
 
 console.log(faPiggyBank.iconName);
 
-ReactDOM.render(<Launch />, document.getElementById('root'));
+ReactDOM.render(<AdminPanel />, document.getElementById('root'));
```

Comment:
We're going to want to show the admin panel if the user goes to url.com/admin. For example, allostatsis.sandboxneu.com/admin. We can use routes to achieve that, allostasis has an example of this already. Let me know if you run into any issues or have any problems.